cdk/app.py

```diff
@@ -1,37 +1,42 @@
 #!/usr/bin/env python3
 from json import dumps as json_dumps
-from sys import argv
+from sys import argv, stdout
 
-import yaml
 from aws_cdk import core
+from ruamel.yaml import YAML, SafeLoader
+from ruamel.yaml import load as yaml_load
 
+from domino_cdk.config import config_loader, config_template
 from domino_cdk.eks_stack import DominoEksStack
 from domino_cdk.util import DominoCdkUtil
 
-app = core.App()
 
-with open(app.node.try_get_context("config") or "config.yaml") as f:
-    y = f.read()
-    cfg = yaml.safe_load(y)
+def main():
+    app = core.App()
 
-env_vars = {}
+    with open(app.node.try_get_context("config") or "config.yaml") as f:
```

Comment:
this bails if config.yaml doesn't exist

Reply:
If you're doing cdk synth, it should bail if config.yaml doesn't exist. If you're doing one of the special args (ie getting the template), then it shouldn't be getting to this part of the code.